fragments/users.js

```diff
@@ -8,6 +8,7 @@ export const ME = gql`
       id
       name
       bioId
+      weblnSats @client
```

Comment:
does it make sense to expose `weblnSats` differently than `sats`? then you probably need to add more conditionals in the code

Reply:
Imo, yes, it does make sense. [Related to this](https://github.com/stackernews/stacker.news/pull/715#discussion_r1437531356).